daemon/logger/splunk/splunk.go

```diff
@@ -363,6 +364,11 @@ func (l *splunkLoggerJSON) Log(msg *logger.Message) error {
 }
 
 func (l *splunkLoggerRaw) Log(msg *logger.Message) error {
+	// empty or whitespace-only messages are not accepted by HEC
+	if strings.TrimSpace(string(msg.Line)) == "" {
```

Comment:
I believe performance at this place is not of sufficient importance to make the logic more complicated:
- most lines will not be empty or whitespace only, so the common case is a line that will be logged
- whenever a line is logged we write data to kernel space. A syscall is probably much more expensive than TrimSpace
- but if you know of a better way of checking "string has at least one non-whitespace character" that would be a nice improvement